scps/SCP-002.md

```diff
@@ -0,0 +1,69 @@
+## SCP-002: Datastore P2P interface
+
+This proposal is a system that allows stores to create custom p2p channels to other peers interested in a common orbitdb datastore.
+
+**Definitions**:
+* Asking peer: The peer initiating the connection
+* Responding peer: The peer responding to the initiation.
+* The network: Cloud of peers all interconnected interested in the datastore in question
+* Broadcast channel: Global pubsub channel of the corresponding datastore.
+* Application specific P2P API: Interface diagrammed by higher level code implementing `orbit-db-store`; Defined by the store type.
+
+### Hypothetical model N1
+> Note all p2p traffic is over IPFS pubsub
+
+**Flow**:
```

Comment:
Note the prior work here:

- https://github.com/ipfs-shipyard/ipfs-pubsub-1on1/
- https://github.com/ipfs-shipyard/ipfs-pubsub-peer-monitor/

Ideally we'd appropriate these (or at least get commit access)